CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+# 1.0.16
```

Comment:
please change to 
```
# Changelog
All notable changes to this project will be documented in this file.

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## [Unreleased]
### Added
- Add support for MWSV2 protocol in Java modules

### Changed
- Renamed com.mdsol.mauth.Signer.generateRequestHeaders to com.mdsol.mauth.Signer.generateRequestHeadersV1
```
and possibly
```
### Deprecated
- //please add headers and V1 support here
```

Reply:
Updated https://github.com/mdsol/mauth-jvm-clients/pull/89/commits/d24aad04e50a64dc504d8035d8a1a56901a81c5e